flyteidl/clients/go/coreutils/extract_literal_test.go

```diff
@@ -124,34 +124,6 @@ func TestFetchLiteral(t *testing.T) {
 		assert.Nil(t, err)
 	})
 
-	t.Run("Generic", func(t *testing.T) {
-		literalVal := map[string]interface{}{
-			"x": 1,
-			"y": "ystringvalue",
-		}
-		var literalType = &core.LiteralType{Type: &core.LiteralType_Simple{Simple: core.SimpleType_STRUCT}}
-		lit, err := MakeLiteralForType(literalType, literalVal)
```

Comment:
This is tested here.

https://github.com/flyteorg/flyte/pull/5763/files#diff-288b2f7bad6b7d9d1e9540d1d0d2d33fd6fac4a3d461d5f0769f15e86a069050R458-R468